src/components/form.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
I suggest writing the name of React Components capitalized.
So the name of this file would be `src/components/Form.js`